src/drivers/wdm/queries/OpaqueMdlUse/OpaqueMdlUse.sarif

```diff
@@ -0,0 +1,627 @@
+{
+  "$schema" : "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Schemata/sarif-schema-2.1.0.json",
```

Comment:
Is this a results file? If so, please name accordingly and describe what this is the results of.

Reply:
Yep, this is the results for the unit tests in driver_snippet.c.  At the moment Abenezer's script is set up to look for a SARIF file with the same name as the query - we can talk about changing the naming style going forward.